vm/vm.go

```diff
@@ -283,6 +284,12 @@ func (vm *VM) Initialize(
 	if err != nil {
 		return err
 	}
+	if vm.config.AcceptorSize > 256 {
+		return errors.New("AcceptorSize should be less than 256")
+	}
+	if vm.config.AcceptedBlockWindow < 1024 {
```

Comment:
Let's set 256 and 1024 as constants at the top of the file

Reply:
Can we change both of these to formatted errors that include the actual provided value, so that it's clearer to the user?

We should also fix the error messages ie. "should be less than or equal to 256" and "should be greater than or equal to 1024". The values 256 and 1024 should be formatted to include the constants set at the top of the file, so that we do not need to update the error string if we change these defaults in the future.

Reply:
Thanks ! Added constants and updated the error messages.

Reply:
Could you include the actual provided value in the error message?

Reply:
Sure, done!